config-templates/google/gcp-compute-persistent-disk-csi-driver/1.7.1/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: kube-system
   annotations:
     version: 1.7.1
-    revision: 4
+    revision: 5
```

Comment:
I thik we can remove this line also
https://github.com/IBM/ibm-satellite-storage/blob/4646fb74a5f2cb06c662d6f46f283b8d4716fe72/config-templates/google/gcp-compute-persistent-disk-csi-driver/1.7.1/deployment.yaml#L791-L793